### PR TITLE
Add custom sound to push notifications by updating brave-alert-lib (CU-10xfkhr)

### PR DESCRIPTION
**Pull request by trisapeace**
## Test Plan
- :heavy_check_mark: Deploy to `chatbot-dev`
- On Android
   - :heavy_check_mark: Custom sound used for push notifications when Alert is in the foreground
   - :heavy_check_mark: Custom sound used for push notifications when Alert App is closed
   - :heavy_check_mark: Custom sound used for push notifications when Alert App is in the background
   - :question: Custom sound stops when the the app is opened (this seems to work when I use the notification to open it, but not when I open it through the normal app)
- On iOS
 :heavy_check_mark: Custom sound used for push notifications when Alert is in the foreground
   - :x: Custom sound used for push notifications when Alert App is closed
   - :x: Custom sound used for push notifications when Alert App is in the background
   - :question: Custom sound stops when the the app is opened

### CODE REVIEW

File: server/package.json [hunk 1]
```diff
@@ -5,7 +5,7 @@
   "main": "server.js",
   "dependencies": {
     "body-parser": "^1.18.3",
-    "brave-alert-lib": "https://github.com/bravetechnologycoop/brave-alert-lib#v6.0.0",
+    "brave-alert-lib": "https://github.com/bravetechnologycoop/brave-alert-lib#td-add-sound-to-push-notifications",
```
Comment (trisapeace) [inline server/package.json:8]: Reminder to self to update this once https://github.com/bravetechnologycoop/brave-alert-lib/pull/30 is merged.

**Comment by trisapeace** [comment]
Task linked: [CU-10xfkhr Backend & Frontend: Alert push notifications sending and handling](https://app.clickup.com/t/10xfkhr)

**Comment by schwarrrtz** [comment]
I tested this on TestFlight with Theresa and the result was that the custom alert sound only plays when the app is in the foreground. The default system sound plays when a notification is received when the app is in the background

**Review by schwarrrtz — APPROVED** [review]
looks good

**Pull request merged**